charts/hivemq-edge/values.schema.json

```
@@ -32,9 +32,15 @@
       "enabled" : {
         "type" : "boolean"
       },
-      "volume" : {
-        "description": "Volume containing the license.",
-        "type" : "object"
+      "volume": {
+        "licenseSecretName" : {
+          "description": "Name of the secret where the license is stored.",
+          "type" : "string"
+        },
+        "licenseSecretKey" : {
```

Same comment regarding the value name
```suggestion
        "secretKey" : {
```